warp10.go

```diff
@@ -1,5 +1,5 @@
 //
-//   Copyright 2016  Cityzen Data
```

Comment:
Hi could you please update the Copyright in your PR?

Copyright 2016-2018 CityzenData; 2016-2019 Warp10 project

Reply:
There is no such entity as 'Warp10 project' The files were authored by Cityzen Data which renamed itself into SenX on october 29 2018, so legally the code authored by Cityzen Data must now be marked as authored by SenX.